code/modules/response_team/ert_outfits.dm

```diff
@@ -483,16 +483,15 @@
 		/obj/item/reagent_containers/spray/cleaner = 1,
 		/obj/item/storage/bag/trash = 1,
 		/obj/item/storage/box/lights/mixed = 1,
-		/obj/item/holosign_creator = 1,
```

Comment:
The holosign was a duplicate here since it's already included in their backpack right?

Reply:
Correct, the full janitor belt always include the holosign projector, and every ERT has the belt.